packages/xstate-wallet/src/ui/selectors.ts

```diff
@@ -31,6 +31,12 @@ export function getConfirmCreateChannelService(
   return applicationWorkflowState.children.invokeCreateChannelConfirmation as any;
 }
 
+export function getChallengeChannelService(
+  applicationWorkflowState: AppWorkflowState
+): Interpreter<any> {
+  return applicationWorkflowState.children.invokeChallengingProtocol as any;
```

Comment:
This value is `undefined` when testing on Storybook, and I'm unsure why. I'm trying to fetch to internal state / context of the `ChallengeMachine`.